docker/bin/install.sh

```diff
@@ -15,33 +15,29 @@
 
 set -eux
 
-# This script decides based on $RELEASE_FROM_CONTEXT and $RELEASE_VERSION whether to reuse, build,
-# or download the binaries we need.
-if [ "$RELEASE_FROM_CONTEXT" = "true" ]; then
-  echo "*** Reusing binaries in the Docker context..."
-  cp /code/zipkin-exec.jar zipkin-exec.jar
-  cp /code/zipkin-slim.jar zipkin-slim.jar
-elif [ "$RELEASE_VERSION" = "master" ]; then
-  echo "*** Building from source..."
-
-  # Defensively avoid arm64+alpine problems with posix_spawn
-  export MAVEN_OPTS="-Djdk.lang.Process.launchMechanism=vfork"
-
-  # Use the same command as we suggest in zipkin-server/README.md
-  #  * Uses mvn not ./mvnw to reduce layer size: we control the Maven version in Docker
-  (cd /code; mvn -T1C -q --batch-mode -DskipTests -Dlicense.skip=true --also-make -pl zipkin-server package)
```

Comment:
Even without the builder, does it hurt to just keep these three lines in the SNAPSHOT branch below instead of the warning message?

Reply:
it hurts because it is never touched and can be wrong or drift and we wouldn't know it. Explicitly unsupporting nesting builds prevents the next person from losing ages trying to get it to work on multi-arch (also laying a bomb for the next maintainer if they succeed). They'll see where it was removed and why.

Reply:
But it's not so bad if it doesn't support multiarch right? If it drifts then when someone notices they can fix it. There seems to me reasonable bang for the buck in these few commands. Up to you though :)

Reply:
you can add it back in a commit, but I think you will be the only user of this. I like the commit log shows how much horror this created.